test/langtools/tools/javac/classfiles/attributes/SourceFile/NoSourceFileAttribute.java

```diff
@@ -52,7 +48,7 @@ public static void main(String[] args) throws Exception {
 
     public void test() throws IOException {
         assertNull(
-                Classfile.of().parse(getClassFile(NoSourceFileAttribute.class).toPath()).findAttribute(Attributes.SOURCE_FILE).orElse(null),
-                "Classfile should have no SourceFile attribute when compiled without debug information.");
+                ClassFile.of().parse(getClassFile(NoSourceFileAttribute.class).toPath()).findAttribute(Attributes.SOURCE_FILE).orElse(null),
+                "ClassFile should have no SourceFile attribute when compiled without debug information.");
```

Comment:
Not sure if replacing the camel case ClassFile in the assertion message is what you wanted here

Reply:
Nice catch, fixed, thanks.